Formula/eslint.rb

```diff
@@ -0,0 +1,22 @@
+require "language/node"
+
+class Eslint < Formula
+  desc "AST-based pattern checker for JavaScript"
+  homepage "https://eslint.org"
+  url "https://registry.npmjs.org/eslint/-/eslint-5.4.0.tgz"
+  sha256 "9ae7c52b2974a0bcc8c969f7ed9444058afd0352fd3c6f7772fae9fac50bf6c5"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    (testpath/".eslintrc.json").write("{}") # minimal config
+    (testpath/"syntax-error.js").write("{}}")
+    # https://eslint.org/docs/user-guide/command-line-interface#exit-codes
+    shell_output("#{bin}/eslint syntax-error.js", 1)
```

Comment:
Can we `assert_match` on the `Unexpected token` message this throws?

Reply:
I guess we can. However, I based this test on [ESLint's own test](https://github.com/eslint/eslint/blob/1e88170fa57656206a39d579fd0c176d414c275b/tests/lib/cli.js#L270-L277) which only checks the exit code.

Reply:
It's just insurance to make sure we're always getting the expected failure.